src/type/p5.Font.js

```diff
@@ -127,10 +127,184 @@ class Font {
+      * — just click and drag with your mouse to see it from all angles!
+      * 
+      * Use the extrude slider to give your letters depth: slide it up, and your 
+      * flat text turns into a solid, multi-dimensional object.
+      * 
+      * You can also choose from various fonts such as "Anton", "Montserrat", or "Source Serif", 
+      * much like selecting fancy fonts in a word processor,
+      * 
+      * The generated model (a Geometry object) can be manipulated further—rotated, scaled, 
+      * or styled with shaders—to create engaging, interactive visual art.
+      *
+      * @param {String} str The text string to convert into a 3D model.
+      * @param {Number} x The x-coordinate for the starting position of the text.
+      * @param {Number} y The y-coordinate for the starting position of the text.
+      * @param {Number} width Maximum width of the text block (wraps text if exceeded).
+      * @param {Number} height Maximum height of the text block.
+      * @param {Object} [options] Configuration options for the 3D text:
+      * @param {Number} [options.extrude=0] The depth to extrude the text. A value of 0 produces 
+      * flat text; higher values create thicker, 3D models.
+      * @param {Number} [options.sampleFactor=1] A factor controlling the level of detail for the text contours.
+      *  Higher values result in smoother curves.
+      * @return {p5.Geometry} A geometry object representing the 3D model of the text.
+      *
+      * @example
+      * <div modernizr='webgl'>
+      * <code>
+      * let font;
+      * let geom;
+      *
+      * async function setup() {
+      *   createCanvas(200, 200, WEBGL);
+      *   fonts = {
+      *     Anton: await loadFont('https://fonts.gstatic.com/s/anton/v25/1Ptgg87LROyAm0K08i4gS7lu.ttf')
+      *   };
+      *
+      *   // Create 3D geometry from text using the Anton font
+      *   geom = fonts['Anton'].textToModel("Hello", 50, 0, { sampleFactor: 2 });
+      *   geom.clearColors();
+      *   geom.normalize();
+      * }
+      *
+      * function draw() {
+      *   background(255);
+      *   orbitControl(); // Enables mouse control to rotate the 3D text
+      *   fill("red");
+      *   strokeWeight(4);
+      *   scale(min(width, height) / 300);
+      *   model(geom);
+      *
+      *   describe('A red non-extruded "Hello" in Anton on white canvas, rotatable via mouse.');
+      * }
+      * </code>
+      * </div>
+      * 
+      * @example
+      * <div modernizr='webgl'>
+      * <code>
+      * let font;
+      * let geom;
+      *
+      * async function setup() {
+      *   createCanvas(200, 200, WEBGL);
+      *   fonts = {
+      *     Anton: await loadFont('https://fonts.gstatic.com/s/anton/v25/1Ptgg87LROyAm0K08i4gS7lu.ttf'),
+      *     Montserrat: await loadFont('https://fonts.gstatic.com/s/montserrat/v29/JTUHjIg1_i6t8kCHKm4532VJOt5-QNFgpCtr6Ew-Y3tcoqK5.ttf'),
```

Comment:
I think we can change this from being an object to just a single font variable since the example doesn't let people switch fonts interactively as the sketch runs. Maybe we can list the other URLs just in comments if users want to try different ones and rerun?